ODIN/chain.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "odin-mainnet-freya",
```

Comment:
The chain name is different than Chain ID.  It's supposed to be the canonical name of a chain, even as a chain id changes.

For example, if the chain id goes from `odin-mainnet-freya` to `odin-mainnet-tyr` in an upgrade, the chain name might still remain, `odinnetwork`

Would you prefer this be `odinnetwork` or `odin`?